azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -325,50 +396,12 @@ def _unblock(self, op, error):
                 op_to_release.error = error
```

Comment:
>op_to_release.error = error [](start = 16, length = 27)

Doesn't need to be done in this PR, but as part of a coherent error story, we might want to chain a different error from the original to indicate this is failing indirectly due to an error in a different op